lib/core/presentation/pages/event_tickets/event_buy_tickets_page/sub_pages/select_tickets_page/widgets/select_ticket_item.dart

```diff
@@ -0,0 +1,112 @@
+import 'package:app/core/presentation/widgets/image_placeholder_widget.dart';
+import 'package:app/core/presentation/widgets/theme_svg_icon_widget.dart';
+import 'package:app/gen/assets.gen.dart';
+import 'package:app/theme/sizing.dart';
+import 'package:app/theme/spacing.dart';
+import 'package:app/theme/typo.dart';
+import 'package:cached_network_image/cached_network_image.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+
+class SelectTicketItem extends StatelessWidget {
+  const SelectTicketItem({
+    super.key,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    final colorScheme = Theme.of(context).colorScheme;
+    return Padding(
+      padding: EdgeInsets.all(Spacing.smMedium),
+      child: Row(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          // image
+          Container(
+            width: Sizing.medium,
+            height: Sizing.medium,
+            decoration: BoxDecoration(
+              borderRadius: BorderRadius.circular(LemonRadius.extraSmall),
+            ),
+            child: ClipRRect(
```

Comment:
You can apply `clipBehavior: Clip.hardEdge` on Container instead of wrapping it with `ClipRRect`